src/ucp/core/ucp_ep.c

```diff
@@ -2097,6 +2097,12 @@ uint64_t ucp_ep_get_tl_bitmap(ucp_ep_h ep)
 
 void ucp_ep_invoke_err_cb(ucp_ep_h ep, ucs_status_t status)
 {
+    if (ep->flags & UCP_EP_FLAG_INVALIDATED) {
```

Comment:
can move this to the 'if' below?

Reply:
can move but then need to set the flag in 2 places and long if harder to read, i would prefer to handle it separately, + most likely we can remove this case after refactoring of err handling in UCT